src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_params.py

```diff
@@ -24,4 +24,4 @@ def get_subscription_id_list(prefix, **kwargs):#pylint: disable=unused-argument
 register_cli_argument('logout', 'username', help='account user, if missing, logout the current active account')
 
 register_cli_argument('account', 'subscription_name_or_id', options_list=('--name', '-n'), help='Name or ID of subscription.', completer=get_subscription_id_list)
```

Comment:
IMO it is strange to have this be aliased to --name if it actually accepts name or ID. I would recommend just using `--subscription`. The account is technically separate from subscription anyways since one account can have many subscriptions (yes?) so technically --name should refer to an account. This one is kind of a gray area.